src/shared/ui/Button.jsx

```diff
@@ -0,0 +1,7 @@
+import styles from "./Button.module.scss";
+
+function Button() {
+  return <button className={styles.button_home}>Begin!</button>;
```

Comment:
can be just styles.button

Reply:
![1](https://github.com/Hreewx/Filmoteka/assets/138201351/5cbe73cb-0770-4438-a764-66671e23f0c0)